core/chains/evm/gas/cmd/arbgas/main.go

```diff
@@ -0,0 +1,75 @@
+// arbgas takes a single URL argument and prints the result of three GetLegacyGas calls to the Arbitrum gas estimator.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/big"
+	"os"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/smartcontractkit/chainlink/core/assets"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/gas"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+func main() {
+	if l := len(os.Args); l != 2 {
+		log.Fatal("Expected one URL argument but got", l-1)
+	}
+	url := os.Args[1]
+	lggr, sync := logger.NewLogger()
+	defer func() { _ = sync() }()
+	lggr.SetLogLevel(zapcore.DebugLevel)
+
+	withEstimator(context.Background(), lggr, url, func(e gas.Estimator) {
+		printGetLegacyGas(e, make([]byte, 10), 500_000, assets.GWei(1))
+		printGetLegacyGas(e, make([]byte, 10), 500_000, assets.GWei(1), gas.OptForceRefetch)
+		printGetLegacyGas(e, make([]byte, 10), max, assets.GWei(1))
+	})
+}
+
+func printGetLegacyGas(e gas.Estimator, calldata []byte, l2GasLimit uint32, maxGasPrice *big.Int, opts ...gas.Opt) {
+	price, limit, err := e.GetLegacyGas(calldata, l2GasLimit, maxGasPrice, opts...)
+	if err != nil {
+		log.Println("failed to get legacy gas:", err)
+		return
+	}
+	fmt.Println("Price:", (*utils.Wei)(price))
+	fmt.Println("Limit:", limit)
+}
+
+const max = 50_000_000
```

Comment:
should be 1 Billion?

Reply:
:shrug: not for the purpose of this script